feature-libs/user/account/components/login-form/login-form-component.service.ts

```diff
@@ -22,14 +26,28 @@ import { tap, withLatestFrom } from 'rxjs/operators';
 
 @Injectable()
 export class LoginFormComponentService {
+  protected authConfigService = inject(AuthConfigService);
+  protected isNewAuthFlow = inject(FeatureConfigService).isEnabled(
+    'authorizationCodeFlowByDefault'
+  );
```

Comment:
`major:` let's inject the whole `FeatureConfigService` in case we need more feature toggles in this file in the future, and also let's change the access modifier to private so customers can't access this service and build their logic on top of that.


```suggestion
  private featureConfigService = inject(FeatureConfigService);
```